PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -108,9 +108,10 @@ public final class DefaultControlsViewController: ContentControlsViewController
     struct State {
         var hasAppeared = false
         var isTransitioning = false
+        var isSeekingInProgress = false
```

Comment:
You could compute `isSeeking` from props

Reply:
I've discussed that with @AndriiMoskvin. We actually have that case only when you interact with seekerView. If seeking comes from props, like if someone seeked from Apple tv remote or any other updates for progress, seek happens immediately. 
It means that I care about interaction with seekerView, but not the fact that seeking is happening.

Reply:
@RomanTysiachnik Ok 👍